tests/org.jboss.reddeer.junit.test/src/org/jboss/reddeer/junit/test/execution/TestMethodShouldRunTest.java

```diff
@@ -56,4 +57,40 @@ public void testShouldNotRun3() {
 	public void testShouldNotRun4() {
 		fail("Test was not supposed to run because @Ignore annotation is presented and run condition was not met.");
 	}
+	
+	@Test
+	@RunIf(conditionClass={ShouldRun.class, ShouldRun.class}, operation=Operation.AND)
```

Comment:
can you please add test without 'operation' variable defined ? to ensure that default is AND.